src/databricks/labs/ucx/source_code/notebooks/cells.py

```diff
@@ -552,10 +552,18 @@ def notebook_path(self) -> Path | None:
         path = self._code[start + 1 :].strip().strip('"').strip("'")
         return Path(path)
 
+    def build_dependency_graph(self, parent: DependencyGraph) -> list[DependencyProblem]:
```

Comment:
why this method was moved? what's the code change?

Reply:
no code change, but I believe it's good practice to have properties before methods

Reply:
don't mix refactoring with feature increments

Reply:
reverted